pkg/daemon/server/handle.go

```diff
@@ -110,6 +105,17 @@ func (cdh *cniDaemonHandler) handleAdd(req *restful.Request, resp *restful.Respo
 	backOffBase := 5 * time.Microsecond
 	retries := 11
 	ipFamily := ipamtypes.ParseIPFamilyFromString(pod.Annotations[constants.AnnotationIPFamily])
+	handledByWebhook := globalutils.ParseBoolOrDefault(pod.Annotations[constants.AnnotationHandledByWebhook], false)
+
+	if !handledByWebhook {
+		_, _, _, ipFamilyStr, err := webhookutils.ParseNetworkConfigOfPodByPriority(context.TODO(), cdh.mgrAPIReader, pod)
```

Comment:
ipFamilyStr maybe empty here, so we have to defaulting it if empty

Reply:
done